app/controllers/twilio/calls_controller.rb

```diff
@@ -1,22 +1,24 @@
 # frozen_string_literal: true
 module Twilio
   class CallsController < ApplicationController
-    before_filter :find_call
```

Comment:
It's a code style preference. I rather do `@call = Call.find(params[:id])` on each action and make the intention of the code more explicit and clearer, than have a before filter just to avoid that line duplication.